installer/templates/new/widgets/text/text.scss

```diff
@@ -5,6 +5,7 @@ $background-color:  #ec663c;
 
 $title-color:       rgba(255, 255, 255, 0.7);
 $moreinfo-color:    rgba(255, 255, 255, 0.7);
+$updatedat-color:   rgba(255, 255, 255, 0.3);
```

Comment:
`$updatedat-color` despite being consistent with `$moreinfo-color` looks like a typo! 
I read it as `update-dat`:

![dat](https://cloud.githubusercontent.com/assets/645514/21593914/42e1cff0-d125-11e6-93e9-575b490d3f49.jpg)

Maybe we should skip the convention just once..